tests/validator/test_validation_graph.py

```diff
@@ -185,3 +197,279 @@ def test_ExpectationValidationGraph_get_exception_info(
 
     assert left_exception in exception_info
     assert right_exception in exception_info
+
+
+@pytest.mark.integration
+def test_parse_validation_graph():
+    class PandasExecutionEngineStub:
```

Comment:
Does it matter that it's a `Pandas` here? If so, can you just use the real execution engine? That is 
```
import ge

ge.execution_engine.PandasExecutionEngine() 
```
instantiates a real execution engine. What does the stub get you here?

Reply:
@billdirks This is in the name of using as little resource as possible to focus on the matter of the test; so we do this -- the execution engine we need is much less than the full `PandasExecutionEngine` instance, but a little bit more than a "Dummy" one.  Thanks.

Reply:
@alexsherstinsky can you explain what the stub actually does?
Why does it need to have it's `__name__` modified like that? Does the test break if you leave it un-modified?

Reply:
No, that seems reasonable. Thanks for the explanation.

The only suggestion I would have is to define it like this.
```python
class PandasExecutionEngineStub:
   __name__ = "PandasExecutionEngine"

```

Reply:
@Kilo59 I tried this before -- and tried it again just now -- it does not work, because the underlying code I mentioned recognizes the class as of the `PandasExecutionEngineStub` type, instead of as `PandasExecutionEngine` type.